Cargo.toml

```diff
@@ -0,0 +1,44 @@
+[workspace]
+members = ["typegen", "description"]
+resolver = "2"
+
+
+[workspace.package]
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2021"
+version = "0.0.1"
+rust-version = "1.70.0"
+license = "Apache-2.0 OR GPL-3.0"
+repository = "https://github.com/paritytech/scale-typegen"
+homepage = "https://www.parity.io/"
+
+
+[workspace.dependencies]
+
+scale-typegen-description = { version = "0.0.1", path = "description" }
+scale-typegen = { version = "0.0.1", path = "typegen" }
+
+# external dependencies
+parity-scale-codec = { version = "3.6.5", features = ["derive"] }
+proc-macro2 = "1.0.69"
+quote = "1.0.33"
+scale-bits = "0.4.0"
+scale-info = { version = "2.10.0", features = [
+    "derive",
+    "bitvec",
+    "decode",
+    "docs",
+    "serde",
+] }
+smallvec = "1.11.2"
+syn = { version = "2.0.38", features = ["full", "extra-traits"] }
+thiserror = "1.0.50"
+prettyplease = "0.2.15"
```

Comment:
Nice! I guess this is the crate that you mentioned that outputs syntax similar to rustfmt